fastestimator/util/google_download_util.py

```diff
@@ -66,3 +68,25 @@ def _download_file_from_google_drive(file_id: str, destination: str) -> None:
                 progress.update(len(chunk))
                 f.write(chunk)
     progress.close()
+
+
+def download_file_from_google_drive(file_id: str, destination: str, max_retries: int = 3) -> None:
+    """Download the data from the Google drive public URL.
+
+    This method will try to download the file for given number of retries till successful.
+
+    Args:
+        file_id: File ID of Google drive URL.
+        destination: Destination path where the data needs to be stored.
+        max_retries: max number of retries.
+    """
+    for _ in range(max_retries):
+        try:
+            time.sleep(2)
+            _download_file_from_google_drive(file_id, destination)
```

Comment:
this checks for file existence before attempting the download right?

Reply:
yes if file exists it would break the look.